### PR TITLE
Add Cancun GraphQL fields

### PR DESCRIPTION
**Pull request by shemnon**
## PR description

Add the fields for Blobs into the GraphQL service.

### CODE REVIEW

File: ethereum/api/src/main/resources/schema.graphqls [hunk 1]
```diff
@@ -199,6 +199,14 @@ type Block {
   withdrawals are unavailable for this block, this field will be null.
   """
   withdrawals: [Withdrawal!]
+
+  """BlobGasUsed is the total amount of gas used by the transactions."""
```
Comment (macfarla) [inline ethereum/api/src/main/resources/schema.graphqls:203]:
```suggestion
  """BlobGasUsed is the total amount of blob gas used by the transactions."""
```

File: ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/graphql/TransactionDataFetcherTest.java [hunk 2]
```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright ConsenSys AG.
```
Comment (macfarla) [inline ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/graphql/TransactionDataFetcherTest.java:2]:
```suggestion
 * Copyright Hyperledger Besu Contributors.
```

**Review by macfarla — COMMENTED** [review]
spotless sad, otherwise changes LGTM

**Review by macfarla — APPROVED** [review]
LGTM other than spotless

File: ethereum/api/src/main/resources/schema.graphqls (hunk 1 above)
Comment (shemnon) [reply ethereum/api/src/main/resources/schema.graphqls:203]: This is a cut and paste from the execution-apis repo, it should be fixed there first - https://github.com/ethereum/execution-apis/pull/468/files#diff-a44507d2ad2482bb3ed863208b9e1e0393069edfdbd686593bac598901f36770R804

File: ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/graphql/TransactionDataFetcherTest.java (hunk 2 above)
Comment (shemnon) [reply ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/graphql/TransactionDataFetcherTest.java]: done

**Pull request merged**